### ISSUE TITLE
codesnippet plugin 404 with CKEDITOR_SERVE_LOCAL = False

### ISSUE THREAD

**Issue by jiamo** [greyli/flask-ckeditor#47]
with 
CKEDITOR_SERVE_LOCAL = False 
CKEDITOR_ENABLE_CODESNIPPET = True
CKEDITOR_PKG_TYPE = 'full'

It can't get codesnippet default.

```
Request URL: https://cdn.ckeditor.com/4.14.0/full/plugins/codesnippet/plugin.js?t=K24B
Request Method: GET
Status Code: 404 
Remote Address: 204.93.150.153:443
Referrer Policy: strict-origin-when-cross-origin
```

**Comment by jiamo** [comment]
replace     
    
     {{ ckeditor.load() }}

with
   
     <script src="https://cdn.ckeditor.com/4.16.0/full-all/ckeditor.js"></script>

works fine.

Does this mean we need option `full-all` in ` CKEDITOR_PKG_TYPE` ?

**Comment by rpmzandwijk** [comment]
I think you are right. According to https://ckeditor.com/cke4/presets-all, the code snippet plugin is only available in the standard-all and full-all versions of the CDN. These 'all' packages are not downloadable, as far as I can see, so we need a mechanism that PKG_TYPE allows standard-all and full-all only when CKEDITOR_SERVE_LOCAL = False. @greyli what do you think?

**Comment by greyli** [comment]
> so we need a mechanism that PKG_TYPE allows standard-all and full-all only when CKEDITOR_SERVE_LOCAL = False

Sounds reasonable, I will make a PR for it.

**Comment by greyli** [comment]
Fixed and released [0.4.6](https://github.com/greyli/flask-ckeditor/releases/tag/0.4.6).
